static/styles/themes/dark-theme.scss

```diff
@@ -681,3 +681,126 @@ textarea.form-control {
         }
     }
 }
+
+body {
+    // Terminal 3 and 4 bit color codes, colors come from https://gogh-co.github.io/Gogh/
```

Comment:
I'm not generally a fan of having large chunks of commented out code: mayyybe

```css
body.palette-dark-pastel {
 non-commented-out: #000;
...
}
```

or something? Or: maybe just leave the comment and link to the other possible choices?

But honestly it's also fine like this...

Reply:
Thank you, I'll remove them when before this PR moves forward :) For now they're here for me to play with, and anyone else interested in taking a look. I've not found a color scheme I love yet.